services/dkp-insights/0.4.1/defaults/cm.yaml

```diff
@@ -0,0 +1,1486 @@
+        allowPrivilegeEscalation: false
+        capabilities:
+          drop:
+            - ALL
+        privileged: false
+        runAsNonRoot: true
+      global:
+        connectionPool:
+          connectionMaxLifetime: 1h
+          maxIdleConnections: 30
+          maxOpenConnections: 50
+        postgresql:
+          postgresqlDatabase: dkp-insights
+          postgresqlPassword: dkp-insights
+          servicePort: 5432
+      persistence:
+        size: 8Gi
+      resources:
+        limits:
+          cpu: 250m
+          memory: 1Gi
+        requests:
+          cpu: 250m
+          memory: 1Gi
+    reforwarder:
+      resources:
+        limits:
+          cpu: 250m
+          memory: 128Mi
+        requests:
+          cpu: 100m
+          memory: 64Mi
+    resolutionCM:
+      resources:
+        limits:
+          cpu: 250m
+          memory: 128Mi
+        requests:
+          cpu: 100m
+          memory: 64Mi
+    seaweedfs:
+      filer:
+        affinity: |
+          podAffinity:
+            preferredDuringSchedulingIgnoredDuringExecution:
+            - weight: 50
+              podAffinityTerm:
+                labelSelector:
+                  matchExpressions:
+                  - key: app
+                    operator: In
+                    values:
+                    - seaweedfs
+                  - key: component
+                    operator: In
+                    values:
+                    - master
+                topologyKey: topology.kubernetes.io/hostname
+          podAntiAffinity:
+            preferredDuringSchedulingIgnoredDuringExecution:
+            - weight: 50
+              podAffinityTerm:
+                labelSelector:
+                  matchExpressions:
+                  - key: app
+                    operator: In
+                    values:
+                    - seaweedfs
+                  - key: component
+                    operator: In
+                    values:
+                    - volume
+                  - key: component
+                    operator: In
+                    values:
+                    - filer
+                topologyKey: topology.kubernetes.io/hostname
+        data:
+          size: 1Gi
+          type: persistentVolumeClaim
+        enablePVC: false
+        enabled: true
+        extraEnvironmentVars:
+          WEED_LEVELDB2_ENABLED: "false"
+          WEED_MYSQL_ENABLED: "false"
+        extraVolumeMounts: |
+          - name: seaweedfs-filer-config
+            mountPath: "/etc/seaweedfs/filer.toml"
+            subPath: "filer.toml"
+        extraVolumes: |
+          - name: seaweedfs-filer-config
+            configMap:
+              name: {{ include "chart.insightsSeaweedFSCMName" . }}
+              items:
+                - key: filer.toml
+                  path: filer.toml
+        grpcPort: 18888
+        initContainers: |
+          - name: init-db
+            image: docker.io/mesosphere/insights:v0.2.3
```

Comment:
just out of curiosity, should this image tag match what's used here: https://github.com/mesosphere/kommander-applications/pull/1322/files#diff-4f759953a5a7257599836148fbf89038b59090ce1f26f7171ddd24a08f72259aR173? 

that would let us ship one less image in the airgapped bundle, but not sure if this was done intentionally or not